i/instana-agent-operator/README.md

```diff
@@ -8,12 +8,13 @@ For platforms other than ppc64le, instana agent in RPM form are available. For p
 
 To build the agent image using the downloaded tarball, please use the following steps:
 
-1. Create a dockerfile in the same directory as the tarball and add the following lines
+1. Create a dockerfile in the same directory as the tarball & docker cli (for ppc) and add the following lines
 
 ```
 FROM registry.access.redhat.com/ubi8/ubi:8.2
 
 COPY instana-agent-linux-ppcle-64bit.tar.gz /root/
+COPY docker /usr/bin/
```

Comment:
@amitsadaphule Its making use of docker daemon from host by mounting it to container. So only adding the cli will do the job here.
Mounting the host docker daemon to container itself is a serious security concern and product team should look to fix it.

Reply:
@awasthan thanks for the details! Can we maybe try the docker-cli installation like [this](https://github.com/ppc64le/build-scripts/blob/master/d/docker/Dockerfiles/19.03.13_centos8/Dockerfile#L9) instead of copying the binary from host?

Reply:
@amitsadaphule I think it should work. When you update the repo, I can validate it.

Reply:
I've raised a PR @ https://github.com/ppc64le/build-scripts/pull/961 to this effect.

Closing this one.